server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -572,3 +572,22 @@ authorizer_provider_class=org.apache.iotdb.db.auth.authorizer.LocalFileAuthorize
 #If OpenIdAuthorizer is enabled, then openID_url must be set.
 
 #openID_url=
+
+# Uncomment following fields to configure the index root directory.
+# For Window platform, the index is as follows:
+# index_root_dir=data\\index
+# For Linux platform
+# If its prefix is "/", then the path is absolute. Otherwise, it is relative.
+# index_root_dir=data/index
+
+# Is index enable
+enable_index=false
+
+# How many threads can concurrently build index. When <= 0, use CPU core number.
+concurrent_index_build_thread=0
+
+# the default size of sliding window used for the subsequence matching in index framework
+default_index_window_range=10
```

Comment:
> I think all configurations are the default value...

An interesting comment. Let's talk about it.
`index_window_range` is necessary for the subsequence matching scenario.
The value of `index_window_range` varies in different index instances and users often specify its value while index creating. The value in the configuration will be adopted only when "index_window_range" is missed.
Thus I think it's better to be named as "default xxx" since its value isn't always used for all indexes.

Reply:
> > I think all configurations are the default value...
> 
> An interesting comment. Let's talk about it.
> `index_window_range` is necessary for the subsequence matching scenario.
> The value of `index_window_range` varies in different index instances and users often specify its value while index creating. The value in the configuration will be adopted only when "index_window_range" is missed.
> Thus I think it's better to be named as "default xxx" since its value isn't always used for all indexes.

Thanks for your detailed explanation! I think it is necessary.